sumfields.php

```diff
@@ -44,6 +44,22 @@ function sumfields_civicrm_navigationMenu(&$menu) {
   _sumfields_civix_navigationMenu($menu);
 }
 
+/**
+ * Implementation of hook_civicrm_buildForm
+ */
+function sumfields_civicrm_buildForm($formName, &$form) {
+  if ($formName == 'CRM_Sumfields_Form_SumFields') {
+
+    // Inject our tpl here
+    $templatePath = realpath(dirname(__FILE__) . "/templates");
+    // dynamically insert a template block in the page
+    CRM_Core_Region::instance('page-body')->add(array(
+      'template' => "{$templatePath}/CRM/Sumfields/Form/SumFields.Custom.tpl"
+      )
+    );
+  }
+}
+
 /**
```

Comment:
Could we just add the javascript to the existing template and do away with this hook? Seems like that would make the code easier to maintain.

Reply:
Makes sense. I will work on that now.